sitetree/static/css/sitetree_bootstrap_submenu.css

```diff
@@ -0,0 +1,49 @@
+// Taken from https://codepen.io/ajaypatelaj/pen/prHjD
```

Comment:
This probably should be minified. What do you think?

Reply:
I think, that we should keep the human-editable version of this file. So I can keep this and include also the minified version, although I don't think it is ideal due to code duplicity in the repo.
The ideal solution would be to compress the file in `setup.py` during packing.

Another possibility is to only suggest using `django-compressor` in the docs without dealing with the minification at all.